src/common/snippets/src/pass/gated_mlp_tokenization.cpp

```diff
@@ -107,22 +110,27 @@ TokenizeGatedMLPSnippets::TokenizeGatedMLPSnippets(const SnippetsTokenization::C
             return false;
         }
 
-        static const auto body_params_count = 5;  // 2xinput + 3x fc
-        static const auto body_result_count = 1;  // one output
-        static const auto reg_group_count = 5;    // upper-bound of possible buffer count
-
-        // TODO [75567]: move this plugin-specific constraint to the plugin callback
-        if (body_params_count + body_result_count + reg_group_count > config.get_data_ptr_gpr_count()) {
-            return false;
-        }
-
-        const auto ordered_ops = ov::NodeVector{fc_gate, fc_up, act, mul, fc_down};
         const bool allow_shared_params = [&]() {
             const auto mm_gate = ov::as_type_ptr<ov::op::v0::MatMul>(fc_gate);
             const auto mm_up = ov::as_type_ptr<ov::op::v0::MatMul>(fc_up);
             OPENVINO_ASSERT(mm_gate && mm_up, "fc_gate and fc_up must have MatMul type");
             return mm_gate->get_transpose_a() == mm_up->get_transpose_a();
         }();
+        // data input (can be shared or not) + 3x matmul weights + result
```

Comment:
```suggestion
        // data 1x input (can be shared or not) + 3x matmul weights + 1x result
```
for better readability

Reply:
Done